README.md

```diff
@@ -2,3 +2,12 @@
 
 `<paper-tooltip>` is a Material Design label that appears when the user hovers
 over an element with the cursor, or focuses the element with the keyboard.
+
+If the target is focusable, the focus and blur event are preferred over the mouse-events.
+So if the cursor is still in an input field, the tooltip stays althought the mouse was moved elsewhere.
+
+## Attributes
+
+* `align` - ['left', 'center', 'right']. Aligns the tooltip accordingly
```

Comment:
yep, target is for, will correct it, is a mistake.

@alignment: the benefit of the computed left/right is, when using a target element which is less wide than the container. A simple CSS align right would put the tooltip to the complete right of the container. This way the tooltip is positioned at the right end of the target.